src/clients/c++/experimental_api_v2/library/common.h

```diff
@@ -40,6 +40,7 @@
 #include <string>
 #include <thread>
 #include <vector>
+#include <algorithm>
```

Comment:
Probably need to run formatter. No need to restart CI after doing that.

Reply:
Done